<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_ViewUser_Transaction</class>
 <widget class="QMainWindow" name="Admin_ViewUser_Transaction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>400</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 32;
background-color: #598bb8;
</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>358</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="Balance_icon">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>money_631180.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Balance_Labe">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>289</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>289</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>24</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TEST</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>342</width>
       <height>176</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="Full_Name">
        <property name="minimumSize">
         <size>
          <width>340</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>289</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Full Name</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="Username_icon">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>profile_7542248.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="UserName_Label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>289</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>289</width>
            <height>27</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>TEST</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="Nationalid_icon">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>id-card_9284909.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Nationalid_Labe">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>289</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>289</width>
            <height>27</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>TEST</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="Email_icon">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>email_4546924.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Email_Label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>289</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>289</width>
            <height>27</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>TEST</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="PhoneNo_icon">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>phone-call_1082233.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="PhoneNo_Labe">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>289</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>289</width>
            <height>27</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>TEST</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>571</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 32;
background-color: #598bb8;
</string>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>471</width>
       <height>46</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Dashboard here</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>991</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 32;
background-color: #598bb8;
</string>
    </property>
    <widget class="QLabel" name="Username_icon_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>931</width>
       <height>180</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>914</width>
        <height>180</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>231</width>
         <height>111</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>65</y>
       <width>921</width>
       <height>1</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>45</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Receiver</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>45</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Sender</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>45</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>45</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>45</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Amount</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>193</width>
       <height>38</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>left_12655291.png</normaloff>left_12655291.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Transaction</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
